provider/equinix/environ.go

```diff
@@ -335,7 +334,7 @@ func (e *environ) StartInstance(ctx context.ProviderCallContext, args environs.S
 	}
 
 	// Render the required tags for the instance.
-	packetTags := []string{}
+	packetTags := make([]string, len(args.InstanceConfig.Tags))
```

Comment:
Same here: 

```
packetTags := make([]string, 0, len(args.InstanceConfig.Tags))
```

Otherwise, you will have `len(args.InstanceConfig.Tags)` empty values followed by the ones you append.

Reply:
I don't see point in allocating packetTags in this specific case. Maybe I should revert it to packetTags := []string{}

Reply:
If you do revert it please change it into `var packetRags []string`. No need to allocate an empty slice; append will take care of this for you.